packages/table-module/__tests__/helper.test.ts

```
@@ -0,0 +1,108 @@
+import * as core from '@wangeditor-next/core'
+import * as slate from 'slate'
+
+import createEditor from '../../../tests/utils/create-editor'
+import { TableCellElement, TableElement } from '../src/module/custom-types'
```

ESLint

[error] 'TableCellElement' is defined but never used.

(@typescript-eslint/no-unused-vars)